x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/expanded_row_json_pane.tsx

```diff
@@ -25,13 +19,9 @@ export const ExpandedRowJsonPane: FC<Props> = ({ json }) => {
       <EuiFlexGroup>
         <EuiFlexItem>
           <EuiSpacer size="s" />
-          <EuiCodeEditor
+          <EuiCodeBlock fontSize="s" language="json" paddingSize="s" style={{ width: '100%' }}>
```

Comment:
You should add an `aria-label`, something like `JSON of transform configuration`

Reply:
Added in 80c039bd37efec8df0619f0e015f4aaba180ca79